exercise/messenger/index.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./assets/css/style.css" />
+    <title>messenger</title>
+  </head>
+  <body>
+    <div id="main">
+      <input type="checkbox" id="input" placeholder="+" />
+      <button class="button__message button--open">+</button>
+      <button class="button__message button--close">x</button>
+      <img
+        class="email__img email__open"
+        src="./assets/img/mail.png"
+        alt="email"
+      />
+
+      <label for="input"></label>
+
+      <div class="messenger">
+        <div class="messager__title">
+          <img
+            class="button__back"
+            src="./assets/img/back-arrow.png"
+            alt="button back"
+          />
+          <div class="title__name">
+            <div class="group__title">Class B1</div>
+            <div class="group__active">4 active now</div>
+          </div>
+          <div class="more">...</div>
```

Comment:
thử đổi nó sang button và ko chứa text như nào xem nhé em
cái button more thì ko dùng ảnh nhé

```suggestion
<div class="messager__header">
          <button class="messager__header-back" />
          <div class="messager__header-information">
            <div class="messager__header-title">Class B1</div>
            <div class="messager__header-status">4 active now</div>
          </div>
          <button class="messager__control--more" />
        </div>
```